qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/DependencyManager.groovy

```diff
@@ -264,7 +264,7 @@ class DependencyManager {
 
         CreateCustomerView createCustomerView
         try {
-            createCustomerView = new CreateCustomerView(this.createCustomerController, this.viewModel, this.createCustomerViewModel)
+            createCustomerView = new CreateCustomerView(this.viewModel, this.createCustomerViewModel, this.createCustomerController)
```

Comment:
Why changing the parameter list here? I fear this could lead to some confusion and merge conflicts in the work of others.

Reply:
Ah I see, you made this change to have the same parameters for the constructor as for the CreateOffer use case. I appreciate your intention but I am not sure if we should do this here. I would at least opt for doing a general refactoring in another PR dedicated to such changes :)

Reply:
I reverted the change as you suggested 👍